README.md

```diff
@@ -0,0 +1,130 @@
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <h3><b>CATALOGUE OF THINGS</b></h3>
+    <a href="https://drive.google.com/file/d/10y3xExClyVJTA6OU4jws-bmI-w9KU-Kx/view?usp=share_link"> Live Demo </a>
+
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [OOP School Library] <a name="about-project"></a>
+
+> Catalogue of things is ruby project used record books, Music Album and games in the library. The library user is have multiple selection of choosing which activity to perfom at the library.
+Depending on wether he want either music, book, or game... he gets different selection on listing all the available items in the store, and few choices are given to add his favorite to the store. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Ruby
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Install Ruby on your computer
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+> A computer with Ruby
```

Comment:
- [ ] It would be a good thing to document properly how to get a local copy of your project in your EADME for other people who might want to check out your project.
- [ ] Also, ensure that you add the step for starting the project locally. It may not be intuitive for all people on how they will start your console application.